src/Simulation/Simulators.Tests/TypeExtensionsTest.cs

```diff
@@ -36,11 +36,24 @@ public void BasicTypes()
         [Fact]
         public void OperationTypes()
         {
-            var op = new QuantumSimulator().Get<Intrinsic.H>();
-            Assert.Equal("H", op.GetNonQubitArgumentsAsString());
+            var simulators = new CommonNativeSimulator[] { 
```

Comment:
This unit test comes out pretty awkward with hard-coding simulator types into the test body; may be good to parameterize to avoid the awkwardness of manual indexing and instantiating twice.